### PR TITLE
add Asset client

### PR DESCRIPTION
**Pull request by pongad**

### CODE REVIEW

File: google-cloud-clients/google-cloud-asset/README.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+Apache 2.0 - See [LICENSE] for more information.
+
+
+[CONTRIBUTING]:https://github.com/GoogleCloudPlatform/google-cloud-java/blob/master/CONTRIBUTING.md
+[code-of-conduct]:https://github.com/GoogleCloudPlatform/google-cloud-java/blob/master/CODE_OF_CONDUCT.md#contributor-code-of-conduct
+[LICENSE]: https://github.com/GoogleCloudPlatform/google-cloud-java/blob/master/LICENSE
+[cloud-platform]: https://cloud.google.com/
+[cloud-asset]: https://cloud.google.com/
+[asset-product-docs]: https://cloud.google.com/
```
Comment (garrettjonesgoogle) [inline google-cloud-clients/google-cloud-asset/README.md]: Maybe file an issue to fix this later if we don't have a URL yet?

File: google-cloud-clients/google-cloud-asset/src/main/java/com/google/cloud/asset/v1beta1/AssetServiceClient.java [hunk 2]
```diff
@@ -0,0 +1,391 @@
...
+  private static final PathTemplate PROJECT_PATH_TEMPLATE =
+      PathTemplate.createWithoutUrlEncoding("projects/{project}");
+
+  /** Formats a string containing the fully-qualified path to represent a project resource. */
+  public static final String formatProjectName(String project) {
+    return PROJECT_PATH_TEMPLATE.instantiate("project", project);
+  }
+
+  /** Parses the project from the given fully-qualified path which represents a project resource. */
+  public static final String parseProjectFromProjectName(String projectName) {
```
Comment (garrettjonesgoogle) [inline google-cloud-clients/google-cloud-asset/src/main/java/com/google/cloud/asset/v1beta1/AssetServiceClient.java]: nooooooooooooo

File: versions.txt [hunk 3]
```diff
@@ -51,6 +52,7 @@ grpc-google-cloud-vision-v1p2beta1:1.22.0:1.22.1-SNAPSHOT
 grpc-google-cloud-vision-v1p3beta1:0.23.0:0.23.1-SNAPSHOT
 grpc-google-cloud-websecurityscanner-v1alpha:0.23.0:0.23.1-SNAPSHOT
 
+proto-google-cloud-asset-v1beta1:0.23.0:0.23.1-SNAPSHOT
 proto-google-cloud-automl-v1beta1:0.23.0:0.23.1-SNAPSHOT
```
Comment (garrettjonesgoogle) [inline versions.txt]: Missing google-cloud-asset?

File: google-cloud-clients/google-cloud-asset/src/main/java/com/google/cloud/asset/v1beta1/AssetServiceClient.java (hunk 2 above)
Comment (pongad) [reply google-cloud-clients/google-cloud-asset/src/main/java/com/google/cloud/asset/v1beta1/AssetServiceClient.java]: ...sigh....

File: google-cloud-clients/google-cloud-asset/README.md (hunk 1 above)
Comment (pongad) [reply google-cloud-clients/google-cloud-asset/README.md]: Done

File: versions.txt (hunk 3 above)
Comment (pongad) [reply versions.txt]: Done

File: google-cloud-clients/google-cloud-asset/src/main/java/com/google/cloud/asset/v1beta1/AssetServiceClient.java (hunk 2 above)
Comment (pongad) [reply google-cloud-clients/google-cloud-asset/src/main/java/com/google/cloud/asset/v1beta1/AssetServiceClient.java]: Done

**Comment by pongad** [comment]
@garrettjonesgoogle PTAL

**Pull request merged**